src/pack-externals.ts

```diff
@@ -90,10 +90,9 @@ function getProdModules(
       !packageJson.dependencies[externalModule.external] &&
       !packageJson.devDependencies[externalModule.external]
     ) {
-      this.options.verbose &&
-        this.serverless.cli.log(
-          `INFO: Runtime dependency '${externalModule.external}' not found in dependencies or devDependencies. It has been excluded automatically.`
-        );
+      this.log.verbose(
```

Comment:
Hmmm 🤔  okay since `log.info = log.verbose` should we make these `verbose` logs now into `debug` logs?

so any `this.options.verbose && log.verbose` into the equivalent `this.log.debug`('this.options.debug && log.debug') ?

I'm happy to still go ahead with this though